server/file.js

```diff
@@ -0,0 +1,29 @@
+const frappe = require("frappejs")
+var formidable = require("formidable")
```

Comment:
Please use either `const` or `let` (to avoid inconsistent codebase)

Use `let` if the variable needs to be reassigned else `const`